classes/class-gf-securesubmit.php

```diff
@@ -1342,24 +1336,15 @@ protected function buildCheckHolder($feed, $submission_data, $entry) {
      *
      * @return \HpsAddress
      */
-    private function buildAddress($feed, $entry) {
+    private function buildAddress($feed, $entry)
+    {
         $address = new HpsAddress();
-        if (in_array('billingInformation_address', $feed['meta'])) {
-            $address->address
-                = $entry[ $feed['meta']['billingInformation_address'] ] . $entry[ $feed['meta']['billingInformation_address2'] ];
-        }
-        if (in_array('billingInformation_city', $feed['meta'])) {
-            $address->city = $entry[ $feed['meta']['billingInformation_city'] ];
-        }
-        if (in_array('billingInformation_state', $feed['meta'])) {
-            $address->state = $entry[ $feed['meta']['billingInformation_state'] ];
-        }
-        if (in_array('billingInformation_zip', $feed['meta'])) {
-            $address->zip = $entry[ $feed['meta']['billingInformation_zip'] ];
-        }
-        if (in_array('billingInformation_country', $feed['meta'])) {
-            $address->country = $entry[ $feed['meta']['billingInformation_country'] ];
-        }
```

Comment:
This seems to be a shared method, used for more than just PayPlan, which is why it shouldn't be hard coded. If PayPlan has an issue with other country values, only PayPlan specific code should overwrite the country value or throw an error to the end-user.